Formula/go@1.14.rb

```diff
@@ -1,9 +1,9 @@
 class GoAT114 < Formula
   desc "Go programming environment (1.14)"
   homepage "https://golang.org"
-  url "https://golang.org/dl/go1.14.7.src.tar.gz"
-  mirror "https://fossies.org/linux/misc/go1.14.7.src.tar.gz"
-  sha256 "064392433563660c73186991c0a315787688e7c38a561e26647686f89b6c30e3"
+  url "https://dl.google.com/go/go1.14.8.src.tar.gz"
```

Comment:
Which one should be used? And can we have an audit to enforce golang or google please

Reply:
hmm, I might use the old url, let me do a quick check.

Reply:
Yeah, the actual location is the one I used.
```
$ curl -I https://golang.org/dl/go1.14.7.src.tar.gz
HTTP/1.1 302 Found
Date: Tue, 01 Sep 2020 19:22:33 GMT
Content-Type: text/html; charset=utf-8
Location: https://dl.google.com/go/go1.14.7.src.tar.gz
```

Reply:
How do we want to enforce the url?

Reply:
I'd say a rubocop that disalows one of the URLs. Which one is used in official documentation?

Reply:
All URLs in https://golang.org/dl/ are of the form https://golang.org/dl/go1.15.1.src.tar.gz, which redirect to their dl.google.com counterparts.

Reply:
So if you go to the website you get the golang.org url and it does an http redirect? Then I think we should stay with the golang.org url.

Reply:
sure, that works for me, I will submit a rubocop check as well.

Reply:
raised a PR, https://github.com/Homebrew/brew/pull/8579